### PR TITLE
Feature/cash bleeders

### PR DESCRIPTION
**Pull request by gbsDojo**
Adding Utils methods;

Adding Pytest for Utils methods;

Testing Pytest workflow with github actions behavior for Pull Requests;

### CODE REVIEW

**Review by tacio — APPROVED** [review]
LGTM

File: .github/workflows/utils_tests.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        architecture: "x64"
+    - name: Cache dependencies
+      uses: actions/cache@v3
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('.github/workflows/utils_requirements.txt') }}
+    - name: Install dependencies
+      run: |
+          python -m pip install --upgrade pip
+          pip install --upgrade --upgrade-strategy eager -r .github/workflows/utils_requirements.txt
```
Comment (tacio) [inline .github/workflows/utils_tests.yml:31]: Why always upgrade pytest instead of fixing its version?

File: .github/workflows/utils_tests.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    - name: Run tests
+      run: |
+        pytest --junitxml=report.xml
```
Comment (tacio) [inline .github/workflows/utils_tests.yml:34]: Is the report retrievable later?

File: setup.py
Comment (tacio) [inline setup.py:1]: Just for the record... Why all the changes in the requirements?

**Review by tacio — APPROVED** [review]
Left some comments, but just some food for thought for future improvements.

File: setup.py
Comment (gbsDojo) [reply setup.py:1]: It turned out that these bibs were unnecessary

**Pull request merged**